src/analyze_includes/parse_source.py

```diff
@@ -56,27 +56,27 @@ def get_includes_from_file(file: Path) -> List[Include]:
       sure only a subset of headers is used for compilation.
     - Include paths utilizing '../' are not resolved.
     """
-    includes = []
+    includes, inside_comment_block = [], False
     with open(file, encoding="utf-8") as fin:
-        inside_comment_block = False
-        for line in fin.readlines():
-            if not inside_comment_block and "/*" in line:
-                if "*/" not in line:
+        for line in fin:
+            line = line.rstrip()
+            i, n, newline = 0, len(line), ""
```

Comment:
Please rename `newline` to `line_without_comments`. newline is a bit misleading due to this name also being used in context of `\n`.